src/main/java/com/moz/qless/Queue.java

```diff
@@ -57,10 +61,11 @@ public int getHeartbeat() throws IOException {
     return Integer.parseInt(heartbeat.toString());
   }
 
-  public int getMaxConcurrency() throws IOException {
-    return Integer.parseInt(
-        this.client.getConfig().get(LuaConfigParameter.MAX_CONCURRENCY)
-        .toString());
+  public Integer getMaxConcurrency() throws IOException {
```

Comment:
It seems asymmetric to allow returning a `null` Integer for `getMaxConcurrency`, but `setMaxConcurrency` only allows a primitive `int` which cannot be `null`.

Is it meaningful for the max concurrency to be `null`? If so, how do you clear the max concurrency?